### PR TITLE
Update README.md in the grid-contrib repository

### PR DESCRIPTION
**Pull request by chenette**
- Update repo description to explain grid-contrib contents vs grid platform software

- Added description of Grid Track and Trace as an example (with links to the docs
  and source)

- Add "How to Contribute" section with links to website's Contributing page,
  #grid channel and mailing list

- Add "More Information" section with links

- Cleaned up "License" section

Signed-off-by: Anne Chenette <chenette@bitwise.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
-# grid-contrib
\ No newline at end of file
+# grid-contrib
+
+The grid-contrib repository contains contributed software for
+[Hyperledger Grid](https://grid.hyperledger.org), a framework for building
+supply chain solutions for blockchain platforms.
+
+This repository is intended for community-contributed software, including
+example middleware tools and reference implementations for smart contracts
```
Comment (davececchi) [inline README.md]: middleware tools? feels like 'domain models' and 'smart contracts' are a better emphasis here. thoughts?

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+We welcome contributions to this repository, either as changes to existing
+software or as new example tools and reference implementations. Please follow
+the [guidelines for
+contributing](https://grid.hyperledger.org/community/contributing/). For more
+information, please join the [#grid discussion
+channel](https://chat.hyperledger.org/channel/grid) or [Hyperledger Grid
+mailing list](https://lists.hyperledger.org/g/grid).
+
+
+## More Information
```
Comment (davececchi) [inline README.md]: mention the contribs Monday meeting here, maybe?

File: README.md (hunk 1 above)
Comment (chenette) [reply README.md]: @davececchi - How does "including example domain models and smart contracts" sound?  

Or would you keep the "reference implementations" piece?  I.e., "including example domain models and reference implementations for smart contracts"?

A third option would be to drop "example": "including domain models and {reference implementations of} smart contracts".

Comment (chenette) [reply README.md]: Follow-up question: Is a domain model the same thing as a data mode"?  (The grid repo README uses the term "data model").

File: README.md [hunk 3]
```diff
@@ -1 +1,52 @@
-# grid-contrib
\ No newline at end of file
+# grid-contrib
+
+The grid-contrib repository contains contributed software for
+[Hyperledger Grid](https://grid.hyperledger.org), a framework for building
+supply chain solutions for blockchain platforms.
```
Comment (jsmitchell) [inline README.md]: s/blockchain/distributed ledger/

File: README.md [hunk 4]
```diff
@@ -1 +1,52 @@
...
+
+This repository is intended for community-contributed software, including
+example middleware tools and reference implementations for smart contracts
+(also called "transaction families"). This software is designed to work with
+the Hyperledger Grid platform software in the [grid
+repository](https://github.com/hyperledger/grid).
+
+For example, the Grid Track and Trace transaction family provides an off-chain
+smart contract that tracks goods as they move through a supply chain. For more
```
Comment (vaporos) [inline README.md]: Remove "off-chain" - that will be inaccurate when deployed w/Sabre.

File: README.md (hunk 2 above)
Comment (vaporos) [reply README.md]: We should put that at "Join the Discussion" link. "Contributors Meeting" != "Meeting about contrib repo."

**Pull request merged**